controllers/comments.controllers.js

```diff
@@ -0,0 +1,9 @@
+const { removeComment } = require("../models/comments.models.js");
+
+exports.deleteComment = (req, res, next) => {
+  const { comment_id } = req.params;
+  removeComment(comment_id)
+    .then(() => {
+        res.status(204).send()})
```

Comment:
This -` res.status(204).send()` - is perfectly valid but you could simply do `res.sendStatus(204)` if you wanted.